frontend/src/components/CountryLink.tsx

```diff
@@ -1,11 +1,12 @@
-import { FC } from "react";
+import { FC, ReactNode } from "react";
 import Link from "next/link";
 import useConfig from "@hooks/useConfig";
 import { getCountrySlug } from "@helpers/getCountryFields";
 
 type TCountryLink = {
   countryCode: string;
   className?: string;
+  children?: ReactNode;
```

Comment:
I don't think we need to declare the children type on the TCountryLink if we are inheriting the FC (functional component) type